UnityProject/Assets/Scripts/UI/Lobby/CharacterSprites.cs

```diff
@@ -65,14 +67,10 @@ private void UpdateReferenceOffset()
 
 		public void UpdateSprite()
 		{
-			if (sprites == null)
-			{
-				// It's possible that UpdateSprite gets called before Awake
-				// so grab the image here just in case that happens
-				sprites = GetComponent<SpriteHandler>();
-			}
-
-			sprites.ChangeSpriteVariant(referenceOffset , NetWork:false);
+			// It's possible that UpdateSprite gets called before Awake
+			// so try to grab the image here just in case that happens
+			if(sprites != null || TryGetComponent(out sprites))
+				sprites.ChangeSpriteVariant(referenceOffset , NetWork:false);
 		}
```

Comment:
NullReferenceException: Object reference not set to an instance of an object
Lobby.CharacterSprites.UpdateSprite () (at /github/workspace/UnityProject/Assets/Scripts/UI/Lobby/CharacterSprites.cs:75)
Lobby.CharacterSprites.Start () (at /github/workspace/UnityProject/Assets/Scripts/UI/Lobby/CharacterSprites.cs:24)